tests/behat/features/capabilities/install/install.feature

```diff
@@ -0,0 +1,48 @@
+@install @stability
+  Feature: Install through web UI
+  Benefit: In order to let our Drupal users install through the interface
+  Role: AN
+  Goal/desire: Install Open Social
+
+  Scenario: I want to install Open Social via the web browser
+    Given I reset the Open Social install
+
+    And I am on "/core/install.php"
+    And I should see "Open Social"
+    And I press the "Save and continue" button
+    And task "Choose language" is done
+
+#    Then I should see "Requirements review"
+#    And I should see "continue anyway"
+#    When I follow "continue anyway"
+    Then task "Verify requirements" is done
+
+#    This is already coming from our settings.php, no need to test it but for clarity:
+#    Then I fill in "edit-mysql-database" with "social"
+#    Then I fill in "edit-mysql-username" with "root"
+#    Then I fill in "edit-mysql-password" with "root"
+#    Then I fill in "edit-mysql-host" with "db"
+#    Then I fill in "edit-mysql-port" with "3306"
+#    Then I submit the form
+    And task "Set up database" is done
+
+    And I should see "Install optional modules"
+    And I should see "Enable additional features"
+    And I should see checked the box "edit-optional-modules-social-group-flexible-group"
+    When I press the "Save and continue" button
+    Then I should see "Installing Open Social"
+    And I wait for the installer to finish
+
+    Then I should see "Configure site"
+    And I fill in "site_name" with "Some site name"
+    And I fill in "site_mail" with "site@example.com"
+    And I fill in "edit-account-name" with "admin"
+    And I fill in "edit-account-pass-pass1" with "password"
+    And I fill in "edit-account-pass-pass2" with "password"
+    And I fill in "edit-account-mail" with "admin@example.com"
+
+    And I uncheck the box "edit-enable-update-status-emails"
+    And I uncheck the box "edit-enable-update-status-module"
+    And I press the "Save and continue" button
+    Then task "Configure site" is done
+    And I am on the homepage
```

Comment:
One of the questions I had, and why I didn't merge it yet before is, does this add the correct value?

We could for example install it with demo content, and run some other tests after this install to ensure not only the Web installer works, but also Open Social itself works correctly after being installed on the web.

Right now we do our testing, and just reinstall. 
Instead of Install through web and run other tests on top of that. 

Curious what the team thinks!

Reply:
Also on a side note, I have been trying out testing this exact scenario (so just installing through the Web) with a Browser test base which runs already locally :) if I can get that working on the CI this will become obsolete

Reply:
@ronaldtebrake Sounds great! If I am not mistaken you are trying to do that in this PR, right https://github.com/goalgorilla/open_social/pull/2970? I think we can postpone this PR until you finish with a Browser test.

Reply:
@ronaldtebrake The most ideal situation of testing will be to repeat the same steps of tests that we are doing now after installation via Drush. This means that 2 x 7 suits will run. IMHO, we should be dependent on Web-based installation more than Drush SI-based installations. So, I would say this is a good start.

Reply:
@ronaldtebrake Could you re-review this, please?

Reply:
@Ressinel I am happy to get this in :) if all is green, good for me!

Reply:
Also closed the other issue for now, don't have time to complete it and we have an issue in Core to test it properly in PHPUnit which will take me a bit more time to get fixed